### PR TITLE
SAN-137-LFP-Unable-to-pay Updating onlinePaymentUnavailable.html

### PR DESCRIPTION
**Pull request by klyttle-ch**
Updating onlinePaymentUnavailable.html

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by rbull-ch** [comment]
Since we are using the existing Controller and Page here - please can you tidy up the method name in the controller.
It looks like the `getPpsNoPenaltyFound` could be renamed to be `getOnlinePaymentUnavailable`?

- https://github.com/companieshouse/penalty-payment-web/blob/master/src/main/java/uk/gov/companieshouse/web/pps/controller/pps/OnlinePaymentUnavailableController.java#L23

File: src/main/resources/templates/pps/onlinePaymentUnavailable.html [hunk 1]
```diff
@@ -16,72 +16,63 @@
...
-                <p class="govuk-body">Email the LFP finance team at Companies House to tell them you've paid.
-                    Include in your email the:</p>
-                <ul class="govuk-list govuk-list--bullet" id="list-requirements-overseas-payments">
-                    <li>company number</li>
-                    <li>company name</li>
-                    <li>reference number on the penalty notice</li>
-                    <li>amount paid</li>
-                    <li>date paid</li>
+
+                <h2 class="govuk-heading-s">Bacs payment</h2>
```
Comment (rbull-ch) [inline src/main/resources/templates/pps/onlinePaymentUnavailable.html:31]: Should this heading be `Bacs payments`?
Also should this heading be `h3` to match prototype WIP - https://sanctions-payment-prototype-0d6d68d6c82d.herokuapp.com/v9/stop-you-need-to-pay-another-way

File: src/main/resources/templates/pps/onlinePaymentUnavailable.html [hunk 2]
```diff
@@ -16,72 +16,63 @@
...
+                        <span class="govuk-!-font-weight-bold">Sort code</span>:
+                        <span id="late-filing-sort-code"
+                              th:text="${@environment.getProperty('penalty.bank-transfer-late-filing.sort-code')}"></span>
+                    </li>
+                    <li>
+                        <span class="govuk-!-font-weight-bold">Account number</span>:
+                        <span id="late-filing-account-number"
+                              th:text="${@environment.getProperty('penalty.bank-transfer-late-filing.account-number')}"></span>
+                    </li>
+                    <li class="govuk-body"><b>Payment reference:</b> Use your penalty reference, so that we know which penalty you've paid.
```
Comment (rbull-ch) [inline src/main/resources/templates/pps/onlinePaymentUnavailable.html:48]: Looks like you have the colon in the wrong place and the html is a bit different for the bold style.
Should it follow how it has been done already here - https://github.com/companieshouse/penalty-payment-web/blob/master/src/main/resources/templates/pps/bankTransferLateFilingDetails.html#L67?

File: src/main/resources/templates/pps/onlinePaymentUnavailable.html [hunk 3]
```diff
@@ -16,72 +16,63 @@
...
+                              th:text="${@environment.getProperty('penalty.bank-transfer-late-filing.swift-code')}"></span>
+                    </li>
+                    <li>
+                        <span class="govuk-!-font-weight-bold">Payment reference</span>: Use your penalty reference, so
+                        that we know which penalty you've paid.
+                        Your payment may be delayed if you use the wrong reference.
+                    </li>
+                </ul>
+
+                <p class="govuk-body"><b>What happens next</b></p>
```
Comment (rbull-ch) [inline src/main/resources/templates/pps/onlinePaymentUnavailable.html:72]: I think this should be `<h3 class="govuk-heading-s">What happens next</h3>`?

File: src/main/resources/templates/pps/onlinePaymentUnavailable.html [hunk 4]
```diff
@@ -16,72 +16,63 @@
...
-                </div>
-                <p class="govuk-body"></p>
-                <h2 class="govuk-heading-m">Paying from an overseas account</h2>
                 <p class="govuk-body">
-                    You can pay through a bank transfer from an overseas account.
+                    You cannot use this online service to pay this penalty.
                 </p>
                 <p class="govuk-body">
-                    You'll need to use Companies House swift code, account number and account name.
+                    Pay by bank transfer instead, using these account details
```
[the remark this answers is not in the document]
Comment (klyttle-ch) [reply src/main/resources/templates/pps/onlinePaymentUnavailable.html:25]: Np full stop in prototype, but will add it in as I think it's needed

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by rbull-ch — APPROVED** [review]

**Review by acallaghan-ch — APPROVED** [review]

**Pull request merged**